### PR TITLE
Adds snippet for restoring a crypto key version

### PR DESCRIPTION
**Pull request by WalterHub**
New snippet to use at https://cloud.google.com/kms/docs/destroy-restore.

### CODE REVIEW

File: kms/src/test/java/com/example/SnippetsIT.java [hunk 1]
```diff
@@ -204,6 +204,24 @@ public void destroyCryptoKeyVersion_destroys() throws Exception {
         KEY_RING_ID, CRYPTO_KEY_ID, version));
   }
 
+
+  @Test
+  public void restoreCryptoKeyVersion_restores() throws Exception {
+    Snippets.createCryptoKeyVersion(PROJECT_ID, LOCATION_ID, KEY_RING_ID, CRYPTO_KEY_ID);
+
+    Matcher matcher = Pattern.compile(".*cryptoKeyVersions/(\\d+)\",\"state\":\"DESTROY_SCHEDULED\".*",
```
Comment (lesv) [inline kms/src/test/java/com/example/SnippetsIT.java]: This might be a bit brittle.  What if it's actually done?

Comment (WalterHub) [reply kms/src/test/java/com/example/SnippetsIT.java]: Do you mean if the destruction already occurred after the wait period?

Comment (lesv) [reply kms/src/test/java/com/example/SnippetsIT.java]: Yes

**Comment by lesv** [comment]
@kurtisvg Sponge looks unhappy - I'm expecting to see a bit more  PTAL

**Comment by WalterHub** [comment]
(fyi, I'm not sure why the UI is indicating I dismissed the comment from @lesv)

**Comment by kurtisvg** [comment]
@lesv Tests for this branch are still WIP. If you want to pull it, I'll take responsibility for fixing any tests it might break.

**Pull request merged**